packages/dashboard/src/components/pageStructure/leftRail.js

```diff
@@ -86,7 +86,12 @@ function getNewTemplatesMetaData(metaDataEntries, days) {
 
 function LeftRail() {
   const { state } = useRouteHistory();
-  const { newStoryURL, version, leftRailSecondaryNavigation } = useConfig();
+  const {
+    newStoryURL,
+    version,
+    leftRailSecondaryNavigation,
+    styleConstants: { topOffset },
```

Comment:
this'll need to get updated for the unit test set up of leftRail (or give a default, that might also work). 
<img width="780" alt="Screen Shot 2022-01-18 at 4 25 04 PM" src="https://user-images.githubusercontent.com/10720454/150035136-cbdb9b07-117e-45e4-8cbc-336e2347dfba.png">

Reply:
Thanks